I1204 04:55:46.662580 4612791808 bigquery_client.py:1144] Requesting discovery document from https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest
I1204 04:55:46.803221 4612791808 model.py:111] --request-start--
I1204 04:55:46.803338 4612791808 model.py:112] -headers-start-
I1204 04:55:46.803397 4612791808 model.py:114] accept: */*
I1204 04:55:46.803450 4612791808 model.py:114] accept-encoding: gzip, deflate
I1204 04:55:46.803498 4612791808 model.py:114] user-agent: google-api-python-client/1.7.10 (gzip)
I1204 04:55:46.803544 4612791808 model.py:115] -headers-end-
I1204 04:55:46.803589 4612791808 model.py:116] -path-parameters-start-
I1204 04:55:46.803636 4612791808 model.py:118] datasetId: weatherData
I1204 04:55:46.803681 4612791808 model.py:118] projectId: test456depp
I1204 04:55:46.803727 4612791808 model.py:119] -path-parameters-end-
I1204 04:55:46.803770 4612791808 model.py:120] body: None
I1204 04:55:46.803865 4612791808 model.py:121] query: ?deleteContents=false
I1204 04:55:46.803912 4612791808 model.py:122] --request-end--
I1204 04:55:46.804025 4612791808 discovery.py:868] URL being requested: DELETE https://bigquery.googleapis.com/bigquery/v2/projects/test456depp/datasets/weatherData?deleteContents=false
I1204 04:55:47.062140 4612791808 model.py:182] --response-start--
I1204 04:55:47.062273 4612791808 model.py:184] vary: Origin, X-Origin, Referer
I1204 04:55:47.062342 4612791808 model.py:184] content-type: application/json; charset=UTF-8
I1204 04:55:47.062402 4612791808 model.py:184] date: Fri, 04 Dec 2020 09:55:47 GMT
I1204 04:55:47.062459 4612791808 model.py:184] server: ESF
I1204 04:55:47.062510 4612791808 model.py:184] cache-control: private
I1204 04:55:47.062558 4612791808 model.py:184] x-xss-protection: 0
I1204 04:55:47.062608 4612791808 model.py:184] x-frame-options: SAMEORIGIN
I1204 04:55:47.062664 4612791808 model.py:184] x-content-type-options: nosniff
I1204 04:55:47.062704 4612791808 model.py:184] alt-svc: h3-29=":443"; ma=2592000,h3-T051=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
I1204 04:55:47.062745 4612791808 model.py:184] transfer-encoding: chunked
I1204 04:55:47.062784 4612791808 model.py:184] status: 404
I1204 04:55:47.062824 4612791808 model.py:184] content-length: 286
I1204 04:55:47.062864 4612791808 model.py:184] -content-encoding: gzip
I1204 04:55:47.062904 4612791808 model.py:186] b'{\n  "error": {\n    "code": 404,\n    "message": "Not found: Dataset test456depp:weatherData",\n    "errors": [\n      {\n        "message": "Not found: Dataset test456depp:weatherData",\n        "domain": "global",\n        "reason": "notFound"\n      }\n    ],\n    "status": "NOT_FOUND"\n  }\n}\n'
I1204 04:55:47.062947 4612791808 model.py:187] --response-end--
